examples/complex_compare_order.py

```diff
@@ -1,16 +1,22 @@
 '''
 Compare order with a validation set
 '''
-
-from examples.setup_client import setup_client
-from rapidata import RapidataClient, Settings, NaiveReferee, CompareWorkflow, ValidationSelection, LabelingSelection, MultiAsset, MediaAsset
-
+from rapidata import (RapidataClient, 
+                      Settings, 
+                      NaiveReferee, 
+                      CompareWorkflow, 
+                      ValidationSelection, 
+                      LabelingSelection, 
+                      MultiAsset, 
+                      MediaAsset,
+                      PromptMetadata)
 
 def new_compare_order(rapi: RapidataClient):
     logo_path = "examples/data/rapidata_logo.png"
     concept_path = "examples/data/rapidata_concept_logo.jpg"
 
-    # configure validation set
+    # Validation set
+    # This will be shown as defined in the ValidationSelection and will make our annotators understand the task better
```

Comment:
below we set ` question="Which logo is the actual Rapidata logo?",` is this intentional or should it match the other example where we ask which logo is better?

Reply:
i see your point but the other question which one is better is subjective. so not a good validation example